lib/screens/home/pages/profile/profile_page.dart

```diff
@@ -220,4 +222,12 @@ class _ProfilePageState extends State<ProfilePage> {
       ),
     );
   }
+
+  Future<void> showloader(BuildContext context) {
```

Comment:
Please change the name to _showProgressIndicator()

Reply:
Done